bin/prepare_project_edition.sh

```diff
@@ -36,15 +39,27 @@ echo "> Move ${DEPENDENCY_PACKAGE_DIR} to ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACK
 mkdir -p ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}
 mv ${DEPENDENCY_PACKAGE_DIR}/* ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}/
 
+# Remove installed dependencies inside the package
+rm -rf ${PROJECT_BUILD_DIR}/${DEPENDENCY_PACKAGE_NAME}/vendor
+
 # Go to main project dir
 cd ${PROJECT_BUILD_DIR}
 
+# Copy auth.json if needed
```

Comment:
This allows us to work with private packages (auth.json is created during these commands:
https://github.com/ezsystems/ezplatform-page-builder/blob/master/.travis.yml#L128-L130 )